snippets/zoe/contracts/test-zoe-hello.js

```diff
@@ -0,0 +1,38 @@
+// TODO: convince prettier that harden() is a global
+/* global harden */
+
+// #region test-imports
+import '@endo/init';
+import { E } from '@endo/far';
+import test from 'ava';
+// #endregion test-imports
+import { start as startState } from '../src/02-state.js';
+import { start as startAccess } from '../src/03-access.js';
+// #region test1
```

Comment:
I can see the source code, so I was able to understand the distinction between `startAccess` and `startState`, but it's not obvious from looking at the generated docs.  Does it make sense to add a comment in the tests where they're called?

Reply:
I re-factored it to use
```js
import * as state from '../src/02-state.js';
```

and then `state.start()`. I hope that's sufficiently clear.